docs/Features/XMLFeatures.md

```diff
@@ -0,0 +1,144 @@
+# XML Features
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides a number of notable features, with demos below.
+<!--  -->
+
+## XML Tag Auto Close
+
+XML tag `auto-close` is supported:
+
+Using `/` in an opening tag will auto close the tag.
+
+![Auto Close Tag Slash](../images/Features/AutoCloseTagSlashXML.gif)
+
+<!-- Using a `>` at the end of an opening tag generates the corresponding closing tag.
+
+![Auto Close Tag End](../images/Features/AutoCloseTagEndXML.gif) -->
+
+<!--  -->
```

Comment:
Remove this comment, or uncomment the content it if you want to include it in the documentation.

Also, there are some empty comments in this file that should be removed.